andela/bad_ass-ah-frontend
# #164489899 Opt in and out of notifications

**What does this PR do?**

- This PR adds the option of opt-in or out of notifications

**Description of Task to be completed?**

- Give a user the ability to opt-in or out of notifications whenever he wants

**How should this be manually tested?**

- Go to the app and log in
- Click on notification icon in the navbar, then you will see `Email Notifications` with a toggle button on the right side
- You can turn them on or off as you want

**What are the relevant pivotal tracker stories?**

[#164489899](https://www.pivotaltracker.com/story/show/164489899)

**Screenshots (if appropriate)**

<img width="774" alt="Screen Shot 2019-06-18 at 12 54 13" src="https://user-images.githubusercontent.com/32579116/59676612-36c6dd80-91c8-11e9-96c8-6c5ecde062cd.png">
